test/runtests.jl

```diff
@@ -11,7 +11,7 @@ using Test
     Random.seed!(1234)
 
     # Generate a set of data from the posterior we want to estimate.
-    data = rand(Normal(0, 1), 300)
+    data = rand(Normal(0., 1), 300)
```

Comment:
I don't see why it would be useful to change this?
```suggestion
    data = rand(Normal(0, 1), 300)
```
The only slightly simpler alternative would be
```suggestion
    data = rand(Normal(), 300)
```

Reply:
this was not meant as a change, i probably accidentally committed this...